.github/workflows/ci.yml

```diff
@@ -2,15 +2,15 @@ name: build
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   test:
     strategy:
       matrix:
-        go: [ '1.15.x', '1.16.x', '1.17.x' ]
+        go: ["1.15.x", "1.16.x", "1.17.x"]
```

Comment:
Why do we want to run with older versions of Go?

Reply:
This is not my code. Forked from swaggo/swag. I have decided to keep it because we would probably want to contribute back to swaggo/swag in the future, and it's helpful to keep up to their standards.